### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mahalrs**

### CODE REVIEW

**Review by nitishm — COMMENTED** [review]
Can you provide a link to the official website > developer docs as well in here - https://azure.github.io/orkestra/developers.html

**Review by nitishm — CHANGES_REQUESTED** [review]
Add docs link

**Comment by mahalrs** [comment]
> Add docs link

I  added a Resources section. Docs link to API reference page on website. Developer's Guide link to Developers page on website. Let me know if you have any suggestions.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,58 +1,67 @@
...
+
+## Support
+
+For questions about building, running, or troubleshooting, start with the [Developer's Guide][dev-guide], and work your way through the process that we've outlined. If that doesn't answer your question(s), try to post on [Discussion][discussion-link] tab or if you think you found a bug, please file an [issue][issue-link].
+
+[dev-guide]: https://azure.github.io/orkestra/developers.html
+[dev-guide-tests]: https://azure.github.io/orkestra/developers.html#testing--debugging
+[code-conduct-link]: https://github.com/Azure/orkestra/blob/main/CODE_OF_CONDUCT.md
+[security-link]: https://github.com/Azure/orkestra/blob/main/SECURITY.md
+[good-issue]: https://github.com/Azure/orkestra/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```
Comment (nitishm) [inline CONTRIBUTING.md:65]:
```suggestion
[good-first-issue]: https://github.com/Azure/orkestra/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

**Review by nitishm — APPROVED** [review]
🚀

**Pull request merged**